universal-application-tool-0.0.1/app/repository/ApplicationRepository.java

```diff
@@ -0,0 +1,50 @@
+package repository;
+
+import static java.util.concurrent.CompletableFuture.supplyAsync;
+
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+import javax.inject.Inject;
+import models.Applicant;
+import models.Application;
+import models.Program;
+
+public class ApplicationRepository {
```

Comment:
suggestion (non-blocking): Seems like we could use a test suite for this class, given the relative complexity of `submitApplication` with the IDs.

Reply:
I kind of think the browser test covers it - I'm reading "non-blocking" here to mean you're okay with that.